steps/maestro-cloud-upload/1.1.0/step.yml

```diff
@@ -0,0 +1,177 @@
+    entry_file: step.sh
+deps:
+  brew:
+  - name: git
+  - name: wget
+  apt_get:
+  - name: git
+  - name: wget
+is_always_run: false
+is_skippable: false
+run_if: ""
+inputs:
+- api_key: ""
+  opts:
+    description: |
+      Maestro Cloud API key
+    is_expand: true
+    is_required: true
+    is_sensitive: true
+    summary: Maestro Cloud API key
+    title: API Key
+- app_file: ""
+  opts:
+    description: |
+      **Android**: `app_file` should point to an x86 compatible APK file. **iOS**: `app_file` should point to an x86 compatible Simulator build packaged in a zip archive.
+    is_expand: true
+    is_required: true
+    summary: App binary to run your Flows against
+    title: App File
+- opts:
+    description: |
+      By default, the action is looking for a .maestro folder with Maestro flows in the root directory of the project.
+       If you would like to customize this behaviour, you can override it with a workspace argument"
+    is_expand: true
+    is_required: false
+    summary: Flow file or directory
+    title: Flow workspace
+  workspace: .maestro
+- opts:
+    description: |
+      "A name will automatically be provided according to the following order:
+        - If it is a Pull Request, use Pull Request title as name
+        - If it is a normal push, use commit message as name
+        - If for some reason the commit message is not available, use the commit SHA as name
+        - If you want to override this behaviour and specify your own name, you can do so by setting the name argument
+    is_expand: true
+    is_required: false
+    summary: Custom upload name
+    title: Upload Name
+  upload_name: $BITRISE_GIT_MESSAGE
+- async: false
+  opts:
+    description: |
+      If you don't want the action to wait until the Upload has been completed as is the default behaviour, set the async argument to true
+    is_expand: true
+    is_required: false
+    summary: Run in async mode
+    title: Async Mode
+- env: ""
+  opts:
+    description: |
+      If you want to pass environment variables along with your upload, add variables separated by a new line or **\n**.
+      i.e VAR1=1\nVAR2=2 ...
+    is_expand: true
+    is_required: false
+    summary: Pass environment variables to your flows
+    title: Environment variables
+- android_api_level: ""
+  opts:
+    description: ""
+    is_expand: true
+    is_required: false
+    summary: 'Set the Android api level the devices should run (Default: 30)'
+    title: Android api level
+- include_tags: ""
+  opts:
+    description: ""
+    is_expand: true
+    is_required: false
+    summary: Run only flows that contain the specified tags (comma separated) i.e
+      dev,pull-request
+    title: Include tags
+- exclude_tags: ""
+  opts:
+    description: ""
+    is_expand: true
+    is_required: false
+    summary: Exclude flows from running with the specified tags (comma separated)
+      i.e pull-request,experimental
+    title: Exclude tags
+- export_format: "false"
```

Comment:
The name of this input seems to be different than what the `title` suggests. I guess this should be `export_test_report`, right?

Reply:
Good observation. It was meant for export format, but we only support `junit` right now so i made it a boolean.
I will update it